src/users/service/users.service.ts

```diff
@@ -1,27 +1,44 @@
-import { Injectable, NotFoundException } from '@nestjs/common';
+import { Injectable, NotFoundException, Param } from '@nestjs/common';
 import { CreateUserDto } from '../dto/create-user.dto';
 import { UpdateUserDto } from '../dto/update-user.dto';
-import { InjectRepository } from '@nestjs/typeorm';
 import { User } from '../entities/user.entity';
 import { Repository } from 'typeorm';
-
-@Injectable()
+import { Profile } from 'src/profiles/entities/profile.entity';
+import { InjectRepository } from '@nestjs/typeorm';
 export class UsersService {
   constructor(
     @InjectRepository(User)
-    private readonly userRepository: Repository<User>
+    private readonly userRepository: Repository<User>,
+
+    @InjectRepository(Profile)
+    private readonly profileRepository: Repository<Profile>,
+
   ) {}
 
   async create(createUserDto: CreateUserDto) {
       const user = this.userRepository.create(createUserDto);
-      return await this.userRepository.save(user);
+
+      await this.userRepository.save(user);
+
+      const getUser = await this.userRepository.findOne({ where: { email: createUserDto.email } });
+
+
+      const perfil = this.profileRepository.create({userId: +getUser.id, description: "default", image: "default"});
+      await this.profileRepository.save(perfil);
+
+      const response = {
+        ...getUser,
+        perfil
+      }
+
+      return response;
   }
 
   async findAll() {
     return await this.userRepository.find();  
   }
 
-  async findOne(id: number) {
+  async findOne(@Param('id') id: number) {
```

Comment:
N odebes recibir parametros en servicios solo en controladores
```suggestion
  async findOne(@Param('id') id: number) {
```